pom.xml

```diff
@@ -44,10 +44,12 @@
 	</properties>
 
 	<dependencies>
+		<!-- reload4j is a drop-in-replacement for log4j version 1 -->
 		<dependency>
-			<groupId>log4j</groupId>
-			<artifactId>log4j</artifactId>
-			<version>1.2.17</version>
+			<groupId>ch.qos.reload4j</groupId>
+			<artifactId>reload4j</artifactId>
+			<version>1.2.22</version>
+			<scope>provided</scope>
```

Comment:
As `reload4j` is using other artifact coordinates than `log4j` it might happen, that projects using the SyslogWriter64k might result in duplicate class dependencies. I'd like to avoid that using the `provided` scope.